### ISSUE TITLE
Изменения в обработке конфига, помощь в виде man

### ISSUE THREAD

**Issue by Vascom** [matimatik/fbless#28]
Прошу добавить некоторые изменения, с которыми приложение будет более удобным.

1. Как я понял, сейчас конфиг читается из домашнего каталога пользователя, а пример конфига просто в fblessrc.example.
Нельзя ли сделать так, чтобы сперва конфиг читался из домашнего каталога, а при его отсутствии из /etc (например /etc/fbless/fblessrc)? Как раз туда можно положить конфиг по умолчанию.

2. Хорошо бы добавить помощь по использованию программы в виде man.

3. Можно ли добавить настройку размера шрифта? Если это есть, то я не нашёл.

**Comment by btimofeev** [comment]
1. Если я правильно помню при отсутствии конфига в домашней директории пользователя программа сама устанавливает дефолтные настройки кодом. Или вы хотите возможность изменять настройки для всех пользователей сразу?

2. Вы можете помочь проекту написав man и прислав Pull request. Просто если его нет, то видимо до сих пор он никому был не нужен.

3. Это делается настройкой шрифта в вашем терминале.

**Comment by Vascom** [comment]
Да, лучше иметь возможность менять настройки для всех. А, при необходимости, каждый поправит под себя уже.

По остальным пунктам - понятно.

**Comment by btimofeev** [comment]
Я не проверял, но скорее всего будет достаточно добавить в файле options.py ниже 24 строки `'/etc/fblessrc',`

**Comment by Vascom** [comment]
Я сделал pull request.
Можно ли fblessrc.example использовать сразу в качестве /etc/fblessrc? Там все настройки как по умолчанию?
